src/onnx/parse_rnn.cpp

```diff
@@ -33,6 +33,29 @@ namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace onnx {
 
+void rnn_transpose_inputs(onnx_parser::node_info& info, std::vector<instruction_ref>& args)
+{
+    std::vector<int64_t> perm{1, 0, 2};
+    args[0] = info.add_instruction(make_op("transpose", {{"permutation", perm}}), args[0]);
+
+    if(args.size() == 6 and not args[5]->is_undefined())
```

Comment:
args.size() is already 6, as we fill upto 6 arguments, before calling this method.

Reply:
@lakhinderwalia  First check is removed.